### ISSUE TITLE
Could not deploy cert manager stable chart with argocd

### ISSUE THREAD

**Issue by dimitarKiryakov** [argoproj/argo-cd#1591]
**Describe the bug**
I am trying to deploy the cert manager stable chart from stable repo without success with argocd.  https://github.com/helm/charts/tree/master/stable/cert-manager

**To Reproduce**
1. Go to https://github.com/helm/charts/tree/master/stable/cert-manager
2. Do the manual steps until 'helm install chart'
3. Execute the following argocd command:
``` 
argocd app create cert-manager \
  --repo https://github.com/helm/charts.git \
  --path stable/cert-manager/ \
  --dest-server https://kubernetes.default.svc \
  --dest-namespace cert-manager
```
4. The execute
```
argocd sync cert-manager
```
5. See error
Argocd is failing with the following error: 
```
kubectl convert is DEPRECATED and will be removed in a future version. In order to convert, kubectl apply the object to the cluster, then kubectl get at the desired version. error: unable to decode "/dev/shm/391372122": no kind "APIService" is registered for version "apiregistration.k8s.io/v1" in scheme "k8s.io/kubernetes/pkg/api/legacyscheme/scheme.go:29"
```
**Expected behavior**
To install the chart successfully

**Comment by alexec** [comment]
Can you confirm which version of Argo CD you are using?

**Comment by alexmt** [comment]
Just reproduced it with latest

**Comment by alexec** [comment]
Is there a workaround for this? E.g. updating your manifests to correct version?

**Comment by alexmt** [comment]
I'm pretty sure this is a duplicate of https://github.com/argoproj/argo-cd/issues/1414 . The fix is available in jus released [v1.0.0-rc3](https://github.com/argoproj/argo-cd/releases/tag/v1.0.0-rc3)
